specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -975,6 +1028,120 @@
     }
   },
   "definitions": {
+    "KeyReleaseCondition": {
+      "properties": {
+        "claim": {
+          "x-ms-client-name": "claim name",
+          "type": "string",
+          "minLength": 1,
+          "description": "claim name"
+        },
+        "condition": {
+          "x-ms-client-name": "condition to test",
+          "type": "string",
+          "minLength": 1,
+          "description": "condition to test",
+          "enum": [
+            "equals"
+          ],
+          "x-ms-enum": {
+            "name": "KeyReleaseCondition",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "equals",
+                "description": "equals comparison."
+              }
+            ]
+          }
+        },
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "KeyReleaseAuthority": {
+      "properties": {
+        "authority": {
+          "x-ms-client-name": "authority name",
+          "type": "string",
+          "minLength": 1,
+          "description": "authority name"
+        },
+        "allOf": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyReleaseCondition",
+            "description": "Required conditions."
+          }
+        }
+      }
+    },
+    "KeyReleasePolicy": {
+      "properties": {
+        "version": {
+          "x-ms-client-name": "version",
+          "description": "authority name",
+          "type": "string",
+          "enum": [
+            "0.2"
+          ]
+        },
+        "anyOf": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyReleaseAuthority",
+            "description": "Authorities."
+          }
+        }
+      }
+    },
+    "ProtectedKey": {
+      "properties": {
+        "alg": {
+          "x-ms-client-name": "key wrapping algorithm",
+          "type": "string",
+          "minLength": 1,
+          "description": "key wrapping algorithm identifier",
+          "enum": [
+            "dir"
+          ],
+          "x-ms-enum": {
+            "name": "ProtectedKeyWrappingAlgorithm",
+            "modelAsString": true,
```

Comment:
As above, doesn't seem this should be extensible.

Reply:
I think any algorithm names would need to be extensible. Supported algorithms tend to change over time. That being said why can't we reuse KeyWrapAlgorithm?

Reply:
As above, this will be extensible, perhaps before GA.

Reply:
Resolving this one and one below. Just with the caveat I mentioned about single-valued enums above. If there is a high likelihood of adding more enum values before GA, will they be added? That would be ideal as to not have to workaround how the generators work in this case.